features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/ButtonCompositionGroupMapper.kt

```diff
@@ -33,13 +33,19 @@ internal class ButtonCompositionGroupMapper(
         var cornerRadius: Number? = null
         var colors: ButtonColors? = null
 
+        // Size of the button must be specified in order to calculate the corner radius,
+        // when [CornerSize] is a percentage.
```

Comment:
In which cases can a `CornerSize` be a percentage (or not)?

Reply:
It totally depends on  the choice of our clients (app developers) how they want to implement UI, either they can use a corner with specific dimension size, so no matter what the size of the button is, the corner keeps same. Either they can use percentage like I show in the screenshot, so the corner radius adapt with button size.

In this case, we must support both of them.